### PR TITLE
VACMS-8016 claro recurring events fixes

### PR DESCRIPTION
**Pull request by tjheffner**
## Description

Relates to #8016. Fixes the merge conflicts caused by my testing updates, fixes the button behavior inside the modal when editing a recurring event series.

PR is targeting the `integration-recrurring-events` branch.

## Testing done

- log in as admin
- create an event node with a recurring event. save the node.
- edit the saved node, edit the event series
- cancel any event
- hover over the buttons and see that the behavior is fixed

## Screenshots
Screen Shot 2022-02-14 at 4 03 00 PM

## QA steps
- log in as admin
- create an event node with a recurring event. save the node.
- edit the saved node, edit the event series
- cancel any event
- hover over the buttons and see that the behavior is fixed

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -238,16 +238,16 @@
...
                 "url": "https://github.com/aws/aws-sdk-php.git",
-                "reference": "2f8a8c885daf45fe9f919312a66a68115d6733b5"
+                "reference": "6082f5df05e5c725a44a310649a922b32bbeec54"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/aws/aws-sdk-php/zipball/2f8a8c885daf45fe9f919312a66a68115d6733b5",
-                "reference": "2f8a8c885daf45fe9f919312a66a68115d6733b5",
+                "url": "https://api.github.com/repos/aws/aws-sdk-php/zipball/6082f5df05e5c725a44a310649a922b32bbeec54",
+                "reference": "6082f5df05e5c725a44a310649a922b32bbeec54",
```
Comment (ethanteague) [inline composer.lock:250]: Is this intentional, @tjheffner ?

File: composer.lock [hunk 2]
```diff
@@ -323,9 +323,9 @@
             "support": {
                 "forum": "https://forums.aws.amazon.com/forum.jspa?forumID=80",
                 "issues": "https://github.com/aws/aws-sdk-php/issues",
-                "source": "https://github.com/aws/aws-sdk-php/tree/3.209.22"
+                "source": "https://github.com/aws/aws-sdk-php/tree/3.209.23"
             },
-            "time": "2022-02-10T19:14:07+00:00"
+            "time": "2022-02-11T19:13:32+00:00"
```
Comment (ethanteague) [inline composer.lock:328]: Is this intentional, @tjheffner ?

File: composer.lock [hunk 3]
```diff
@@ -6437,26 +6437,26 @@
...
+                "reference": "3.1.1"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://ftp.drupal.org/files/projects/hierarchy_manager-3.1.0.zip",
-                "reference": "3.1.0",
-                "shasum": "c9facc5ea8582cca61b8ed1bbace507995887401"
+                "url": "https://ftp.drupal.org/files/projects/hierarchy_manager-3.1.1.zip",
+                "reference": "3.1.1",
+                "shasum": "ecb44f021051333893d7422f410b91043867d2dd"
```
Comment (ethanteague) [inline composer.lock:6450]: Is this intentional, @tjheffner ?

File: composer.lock [hunk 4]
```diff
@@ -6437,26 +6437,26 @@
...
             "require": {
                 "drupal/core": "^8 || ^9"
             },
             "type": "drupal-module",
             "extra": {
                 "drupal": {
-                    "version": "3.1.0",
-                    "datestamp": "1633693423",
+                    "version": "3.1.1",
+                    "datestamp": "1642047456",
```
Comment (ethanteague) [inline composer.lock:6459]: Is this intentional, @tjheffner ?

File: composer.lock [hunk 5]
```diff
@@ -8800,17 +8800,17 @@
...
+                "reference": "8.x-1.9"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://ftp.drupal.org/files/projects/pathauto-8.x-1.8.zip",
-                "reference": "8.x-1.8",
-                "shasum": "ede3216abb9c4f77709338d9147334c595046329"
+                "url": "https://ftp.drupal.org/files/projects/pathauto-8.x-1.9.zip",
+                "reference": "8.x-1.9",
+                "shasum": "f075ebff595c9b8b62333d65ad29020330e0ea9d"
```
Comment (ethanteague) [inline composer.lock:8813]: Is this intentional, @tjheffner ?

File: composer.lock [hunk 6]
```diff
@@ -8823,8 +8823,8 @@
             "type": "drupal-module",
             "extra": {
                 "drupal": {
-                    "version": "8.x-1.8",
-                    "datestamp": "1588103046",
+                    "version": "8.x-1.9",
+                    "datestamp": "1644822949",
                     "security-coverage": {
```
Comment (ethanteague) [inline composer.lock:8828]: Is this intentional, @tjheffner ?

File: composer.lock [hunk 7]
```diff
@@ -238,16 +238,16 @@
         },
         {
             "name": "aws/aws-sdk-php",
-            "version": "3.209.22",
+            "version": "3.209.23",
             "source": {
```
[the remark this answers is not in the document]
Comment (tjheffner) [reply composer.lock:242]: All updates to the lock file are intentional, from pulling in the most updated dependencies present in the `main` branch (to fix the conflicts caused by my testing PR). this PR targets the integration-recurring-events branch which doesn't have those updates yet. the diff between that branch and main shouldn't show these after this is merged into the integration branch.

https://github.com/department-of-veterans-affairs/va.gov-cms/pulls?q=is%3Apr+is%3Aclosed+bump

**Comment by swirtSJW** [comment]
Seems like there is lots of unexpected stuff in here.   I think we want to just rebase the integration branch on main rather than bring in a bunch of commits by merge.

**Comment by swirtSJW** [comment]
@tjheffner I rebased the integration branch which had some conflicts.  On the conflicts I resolved them and chose what was the more recent coming in with "main"   @ethanteague I may have broken something by making that choice but I don't think so ????

Tanner this has some conflicts now as well.  Sorry.  I am going to let you resolve them as I am less certain about what needs to stay and what needs to go.

I saw this in your commit history
image
 It is sometimes (not always) the sign of a less than ideal rebase process.  Ideally if you are resolving a conflict on a rebase locally, you should fix the conflict in the code, then add but not commit it.  The added file(s) gets handled by `git rebase --continue`

**Pull request closed without merging**